### PR TITLE
[PXCT-198] MicroPython New Documentation

### PR DESCRIPTION
**Pull request by pedromsousalima**
## What This PR Changes
- New documentation for micropython

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/micropython/03.micropython/02.environment/02.file-system/file-system.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+## Accessing the MicroPython File System
+
+To interact with the MicroPython file system, we’ll use Arduino Labs for MicroPython, which provides tools to manage and view files directly on our device. Here’s how to get started:
+
+1. **Connect to Your Device**: Open Arduino Labs for MicroPython and establish a connection to our microcontroller.
+2. **Upload and Download Files**: Use the file manager to upload files from our computer to the microcontroller or download files back to our computer.
+3. **Organize Files**: We can create folders and store multiple files, making it easy to organize our project.
+
+![Accessing the file system.]()
```
Comment (karlsoderby) [inline content/micropython/03.micropython/02.environment/02.file-system/file-system.md:29]: Missing image here @pedromsousalima

File: content/micropython/03.micropython/02.environment/02.file-system/file-system.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+3. Check the REPL, we should see:
+
+    ```bash
+    This runs from main.py # executed from main.py
+    This runs from my_new_script.py #executed from my_new_script.py
+    ```
+
+Essentially, this is how [modules]() work. You import a module, and use a function from that module.
+
+![Import code from a script.]()
```
Comment (karlsoderby) [inline content/micropython/03.micropython/02.environment/02.file-system/file-system.md:77]: missing image here @pedromsousalima

File: content/micropython/03.micropython/01.basics/00.digital-io/digital-io.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
+---
+title: 'Digital I/O'
+description: 'A guide to digital inputs and outputs using MicroPython.'
+author: 'Pedro Lima'
+tags: [MicroPython, Digital I/O]
+---
+
+Digital pins are fundamental for interacting with the physical world using your Arduino board. With them, you can:
+
+- Control outputs, such as turning an LED on and off.
```
Comment (g-castronovo) [inline content/micropython/03.micropython/01.basics/00.digital-io/digital-io.md:10]:
```suggestion
- Control outputs, such as turning a LED on and off.
```

File: content/micropython/03.micropython/01.basics/00.digital-io/digital-io.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+### Software Requirements
+
+- [Arduino Lab for Micropython](https://labs.arduino.cc/en/labs/micropython) - Arduino Lab for MicroPython is an editor where we can create and run MicroPython scripts on our Arduino board.
+
+***Note that the editor is also available online, at [Arduino Cloud - Arduino Labs for MicroPython](https://lab-micropython.arduino.cc/)***
+
+## Board and Editor Setup
+
+1. Open the [Arduino Lab for MicroPython]() application.
```
Comment (g-castronovo) [inline content/micropython/03.micropython/01.basics/00.digital-io/digital-io.md:63]: Missing url

**Review by karlsoderby — APPROVED** [review]

**Pull request merged**